actions/admin/contact.go

```diff
@@ -182,6 +214,39 @@ func (a *Action) contactsAccept(c *gin.Context) {
 	c.JSON(http.StatusOK, contract)
 }
 
+// contactsConfirm will perform Confirm action on contact with the given id
+// Perform confirm action on contact godoc
+// @Summary		Confirm contact
+// @Description Confirm contact
+// @Tags		Admin
+// @Produce		json
+// @Param		id path string false "Contact id"
+// @Success		200 {object} models.Contact "Changed contact"
+// @Failure		400	"Bad request - Error while getting id from path"
+// @Failure		404	"Not found - Error while getting contact by id"
+// @Failure		422	"Unprocessable entity - Incorrect status of contact"
+// @Failure 	500	"Internal server error - Error while updating contact"
+// @Failure 	500	"Internal server error - Error while changing contact status"
+// @Router		/v1/admin/contact/confirmed/{id} [patch]
+// @Security	x-auth-xpub
+func (a *Action) contactsConfirm(c *gin.Context) {
```

Comment:
I can imagine the situation when the user calls to the admins because he is not able to do it by himself...for some reason; so an admin goes to the admin panel and do the confirmation on behalf of the user 🤔

Reply:
For me, it's total overpower for the admin.
Because confirmation of the contact, makes it marked as verified. Which should only happen with TOTP exchange.

Reply:
Admins like to have power 🙈 
However, if you think that way, and moreover, it wasn't a business decision to add feature like that, I agree with you, we should not allow admins to confirm contacts👍

Reply:
I understand that Admins like to have power.
But actually we have an issue with the confirm contact that we don't have solution for.
Because whole logic of confirmation is done on the client side, we don't have a power to ensure that everything is correct at the server side, so we must just trust the client that he knows what he is doing. Therefore I would like to rather limit the number of clients that we need to trust in this situation.

Reply:
Removed endpoint